basiclti/basiclti-tool/src/webapp/vm/lti_tool_edit.vm

```diff
@@ -0,0 +1,393 @@
+<div class="portletBody">
+    <script type="text/javascript" src="/library/js/headscripts.js"></script>
+    <script>
+        includeLatestJQuery('lti_tool_edit.vm');
+        includeWebjarLibrary('fontawesome-iconpicker');
+    </script>
+    <h3>$tlang.getString("tool.put")</h3>
+
+    #if ($alertMessage)<div class="sak-banner-error">$tlang.getString("gen.alert") $formattedText.escapeHtml($alertMessage)</div>#end
+
+    <form action="#toolForm("")" method="post" name="customizeForm" >
+        #if ($isEdit && $isAdmin && $autoStartUrl )
+            <div style="float:right" id="lti-advantage-div">
+                <a href="$autoStartUrl" class="btn btn-primary" target="sakai-basiclti-admin-iframe" title="$tlang.getString("tool.lti13.auto.title")"
+                   role="button" onclick="showIframe(this.title, false);">$tlang.getString("tool.lti13.auto.button")
+                </a>
+            </div>
+        #end
+
+        #if ($canDeploy)
+            <div style="float:right; display: none;" id="lti-deployment-div">
+                <a href="$deployUrl" class="btn btn-primary" target="sakai-basiclti-admin-iframe" title="$tlang.getString("deploy.tool.site.title")"
+                   role="button" onclick="showIframe(this.title, false);">$tlang.getString("deploy.tool.site.button")
+                </a>
+            </div>
+        #end
+
+        $formInput
+
+        <input type="hidden" name="sakai_csrf_token" value="$sakai_csrf_token" />
+
+        <p class="act">
+            <input type="submit" accesskey ="s" class="active" name="$doToolAction"
+                value="$tlang.getString('gen.save')" />
+            <input type="submit" accesskey ="x" name="$doCancel" value="$tlang.getString('gen.cancel')"
+                 onclick="location = '$sakai_ActionURL.setPanel("ToolSystem")';return false;">
+        </p>
+    </form>
+
+    #if (($isEdit && $isAdmin && $autoStartUrl) || $canDeploy)
+        <div id="modal-iframe-div" style="display:none" tabindex="-1" role="dialog">
+            <iframe name="sakai-basiclti-admin-iframe" id="sakai-basiclti-admin-iframe" src="/library/image/sakai/spinner.gif" style="min-height: 80vh;" tabindex="0"></iframe>
+        </div>
+
+        <script>
+        //<![CDATA[
```

Comment:
looks like you can remove this?

Reply:
Are you saying that the //<![CDATA[ is no longer needed for page validation?